lib/vehicle.rb

```diff
@@ -5,14 +5,17 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
+              :engine, 
+              :plate_type
 
   def initialize(vehicle_details)
     @vin = vehicle_details[:vin]
     @year = vehicle_details[:year]
     @make = vehicle_details[:make]
     @model = vehicle_details[:model]
     @engine = vehicle_details[:engine]
+    @plate_type = plate_type
```

Comment:
This is an interesting way to code this actually, you are running the method of `plate_type` right on initialization and I think that's a great way to do this! I might change the name of the method to `set_plate_type` though, since our method names should be verbs. Great job. 

We would NOT want to do this for line 18, `@registration_date` however, since we should not be setting the registration date until the vehicle is registered.